Framework/DataHandling/inc/MantidDataHandling/SaveOpenGenieAscii.h

```diff
@@ -33,44 +33,65 @@ class DLLExport SaveOpenGenieAscii : public Mantid::API::Algorithm {
   }
 
 private:
+  /// Typedef of a tuple containing the name, type and value as strings
+  using outputTuple = std::tuple<std::string, std::string, std::string>;
```

Comment:
Type names should being with a capital. It would also be better if this was more descriptive, what is an "Output Tuple"? See: http://llvm.org/docs/CodingStandards.html